app/src/main/java/com/infomaniak/mail/utils/LogoutUser.kt

```diff
@@ -78,7 +78,7 @@ class LogoutUser @Inject constructor(
         appContext.getInfomaniakLogin().deleteToken(
             okHttpClient = HttpClient.okHttpClientNoTokenInterceptor,
             token = apiToken,
-            onError = { SentryLog.e("DeleteTokenError", "API response error: $it") },
+            onError = { SentryLog.i("DeleteTokenError", "API response error: $it") },
```

Comment:
See with @LunarX for this one.

Reply:
Moved to https://github.com/Infomaniak/android-kMail/pull/2037